src/components/ReportIt.vue

```diff
@@ -1,90 +1,321 @@
+                    <div>
+                        <label for="emailorphone"
+                            >Please provide your email or phone number:
+                            (Required*)</label
+                        >
+                        <div class="field">
+                            <input
+                                type="text"
+                                v-model="formData.email"
+                                ref="emailorphone"
+                                name="emailorphone"
+                                required
+                            />
+                        </div>
+                    </div>
+                    <br />
+                    <div>
+                        <label for="verificationcode">We sent you a code</label>
+                        <div class="field">
+                            <input
+                                type="text"
+                                ref="verificationcode"
+                                v-model="formData.verification_code"
+                                name="verificationcode"
+                            />
+                        </div>
+                    </div>
+                    <br /><br />
+                    <div>
+                        <div class="ui small info message">
+                            Our conflict resolution/hygiene team will reach out
+                            to you if need be, within 10-28 business days.
+                        </div>
+                    </div>
+                    <hr />
+                    <div class="ui success message" v-if="report_success">
+                        <i class="close icon"></i>
+                        <div>Your report was submitted.</div>
+                    </div>
+                    <div class="ui success message" v-if="report_success">
+                        <i class="close icon"></i>
+                        <div>Your report was submitted.</div>
+                    </div>
+
+                    <div class="ui negative message" v-if="report_failed">
+                        <i class="close icon"></i>
+                        <div>Error sending report.</div>
+                    </div>
+                    <div class="">
+                        <div class="ui input">
+                            <button
+                                class="ui tbb button"
+                                v-bind:class="{ loading: is_loading }"
+                            >
+                                Submit
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </div>
 </template>
 
 <script>
+import store from "@/store";
+
 export default {
-	name: "ReportIt",
-	props: {
-		numbers: Array
-	},
-	data() {
-		return {
-			title: "Use this tool if you think this recipe is any of the following; inappropriate, unauthentic, not original, stolen or a duplicate."
```

Comment:
Looks like you have to add this back. It's used to render the title of this compoent ln 5